subnet/deployment-generator/script/config_gen.js

```diff
@@ -0,0 +1,175 @@
+const crypto = require('crypto');
```

Comment:
All comparation in js, best practice use `===` matching with type.

Reply:
Thanks, updated